gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-edit-page/api-documentation-v4-edit-page.component.ts

```diff
@@ -226,14 +246,21 @@ export class ApiDocumentationV4EditPageComponent implements OnInit, OnDestroy {
           referenceId,
           referenceType: 'GROUP',
         }));
-        return this.apiDocumentationService.updateDocumentationPage(this.api.id, this.activatedRoute.snapshot.params.pageId, {
+
+        const updateDocumentation: EditDocumentation = {
           ...page,
           name: formValue.stepOne.name,
           visibility: formValue.stepOne.visibility,
           content: formValue.content,
           excludedAccessControls: formValue.stepOne.excludeGroups,
           accessControls: [...nonGroupAccessControls, ...selectedGroupAccessControls],
-        });
+          ...(formValue.source && { source: this.obtainSource(formValue.source, formValue.sourceConfiguration) }),
```

Comment:
You can add the logic here like in the create method --> only add `source: XX` when the `formValue.source === 'HTTP'`

Reply:
Done